src/Sulu/Bundle/ContactBundle/Controller/AbstractMediaController.php

```diff
@@ -137,6 +137,54 @@ protected function removeMediaFromEntity($entityName, $id, $mediaId)
     }
 
     /**
+     * Sets the medias which are associated with an entity to the given mediaIds.
+     *
+     * @param string $entityName
+     * @param string $id
+     * @param string $mediaIds
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    protected function setMediasOfEntity($entityName, $id, $mediaIds)
+    {
+        try {
+            $em = $this->getDoctrine()->getManager();
+            $entity = $em->getRepository($entityName)->find($id);
+            $medias = $this->container->get('sulu.repository.media')->findById($mediaIds);
+
+            if (!$entity) {
+                throw new EntityNotFoundException($entityName, $id);
+            }
+
+            $mediaEntityName = $this->getParameter('sulu.model.media.class');
+
+            $foundMediaIds = array_map(function ($media) {
+                return $media->getId();
+            }, $medias);
+            if ($missingMediaIds = array_diff($mediaIds, $foundMediaIds)) {
+                throw new EntityNotFoundException($mediaEntityName, $missingMediaIds[0]);
+            }
+
+            $entity->getMedias()->clear();
+            foreach ($medias as $media) {
+                $entity->addMedia($media);
+            }
+
+            $em->flush();
+
+            $view = $this->view(null, 200);
+        } catch (EntityNotFoundException $enfe) {
+            $view = $this->view($enfe->toArray(), 404);
+        } catch (RestException $exc) {
+            $view = $this->view($exc->toArray(), 400);
+        } catch (\Exception $e) {
+            $view = $this->view($e->getMessage(), 400);
+        }
+
+        return $this->handleView($view);
```

Comment:
Probably, as explained in https://github.com/massiveart/husky/pull/700#discussion_r73294951